app/scripts/views/NodeViews/CodeBlock.js

```diff
@@ -20,12 +20,13 @@ define(['backbone', 'ThreeCSGNodeView'], function (Backbone, ThreeCSGNodeView) {
 
             this.$el.on('mouseup', adjustElements.bind(this));
             this.$el.on('mousemove', adjustElements.bind(this));
+        },
 
+        setCodeblockDraggableOption: function() {
             // Get a value of the cancel option before it is set to avoid losing previous value
             var cancelOption = this.$el.draggable( 'option', 'cancel' );
             if(!cancelOption.match(/.code-block-input/i))
                 this.$el.draggable( 'option', 'cancel', cancelOption + ',.code-block-input' );
-            
```

Comment:
this code should be executed only after `makeDraggable`. `makeDraggable` isn't called anymore in based constructor so move it out in separate method and now it's called in [`makeDraggable`]( https://github.com/pboyer/flood/pull/128/files#diff-89cdc10f977f34bb1ee1f6ac955d076bR618) after proper draggable related initializations